code/modules/research/xenobiology/crossbreeding/_potions.dm

```diff
@@ -0,0 +1,211 @@
+	M.faction |= "[REF(user)]"
+	M.apply_status_effect(STATUS_EFFECT_INLOVE, user)
+	qdel(src)
+
+//Pressure potion - Charged Dark Blue
+/obj/item/slimepotion/spaceproof
+	name = "slime pressurization potion"
+	desc = "A potent chemical sealant that will render any article of clothing airtight. Has two uses."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "potblue"
+	var/uses = 2
+
+/obj/item/slimepotion/spaceproof/afterattack(obj/item/clothing/C, mob/user, proximity)
+	. = ..()
+	if(!uses)
+		qdel(src)
+		return
+	if(!proximity)
+		return
+	if(!istype(C))
+		to_chat(user, "<span class='warning'>The potion can only be used on clothing!</span>")
+		return
+	if(C.min_cold_protection_temperature == SPACE_SUIT_MIN_TEMP_PROTECT && C.clothing_flags & STOPSPRESSUREDAMAGE)
+		to_chat(user, "<span class='warning'>The [C] is already pressure-resistant!</span>")
+		return ..()
+	to_chat(user, "<span class='notice'>You slather the blue gunk over the [C], making it airtight.</span>")
+	C.name = "pressure-resistant [C.name]"
+	C.remove_atom_colour(WASHABLE_COLOUR_PRIORITY)
+	C.add_atom_colour("#000080", FIXED_COLOUR_PRIORITY)
+	C.min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT
+	C.cold_protection = C.body_parts_covered
+	C.clothing_flags |= STOPSPRESSUREDAMAGE
+	uses--
+	if(!uses)
+		qdel(src)
+
+//Enhancer potion - Charged Cerulean
+/obj/item/slimepotion/enhancer/max
+	name = "extract maximizer"
+	desc = "An extremely potent chemical mix that will maximize a slime extract's uses."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "potpurple"
+
+//Lavaproofing potion - Charged Red
+/obj/item/slimepotion/lavaproof
+	name = "slime lavaproofing potion"
+	desc = "A strange, reddish goo said to repel lava as if it were water, without reducing flammability. Has two uses."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "potred"
+	resistance_flags = LAVA_PROOF | FIRE_PROOF
+	var/uses = 2
+
+/obj/item/slimepotion/lavaproof/afterattack(obj/item/C, mob/user, proximity)
+	. = ..()
+	if(!uses)
+		qdel(src)
+		return ..()
+	if(!proximity)
+		return ..()
+	if(!istype(C))
+		to_chat(user, "<span class='warning'>You can't coat this with lavaproofing fluid!</span>")
+		return ..()
+	to_chat(user, "<span class='notice'>You slather the red gunk over the [C], making it lavaproof.</span>")
+	C.name = "lavaproof [C.name]"
+	C.remove_atom_colour(WASHABLE_COLOUR_PRIORITY)
+	C.add_atom_colour("#800000", FIXED_COLOUR_PRIORITY)
+	C.resistance_flags |= LAVA_PROOF
+	if (istype(C, /obj/item/clothing))
+		var/obj/item/clothing/CL = C
+		CL.clothing_flags |= LAVAPROTECT
+	uses--
+	if(!uses)
+		qdel(src)
+
+//Revival potion - Charged Grey
+/obj/item/slimepotion/slime_reviver
+	name = "slime revival potion"
+	desc = "Infused with plasma and compressed gel, this brings dead slimes back to life."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "potsilver"
+
+/obj/item/slimepotion/slime_reviver/attack(mob/living/simple_animal/slime/M, mob/user)
+	if(!isslime(M))
```

Comment:
Would that be too powerful...? I mean, I suppose it does decrease maximum HP.

Reply:
Opted not to - an instant revive just because you're a slime person seems a little excessive, especially when one of the slimepeople races already have infinite extra lives.